docs/src/content/en/reference/tools/mcp-server.mdx

```diff
@@ -0,0 +1,174 @@
+---
```

Comment:
I think this is fine to start but you should put in a follow-up breaking this out into one file for the class, and one for each function.